RELEASE_NOTES.md

```diff
@@ -1,5 +1,90 @@
 # Release Notes
 
+## stable-1.18-3.0.0
+
+- Ambassador
+  - added Ambassador addon ([#524](https://github.com/mesosphere/kubernetes-base-addons/pull/524), [@shaneutt](https://github.com/shaneutt))
+
+- Cert-manager:
+  - v1 API
+  - Renaming our API group from certmanager.k8s.io to cert-manager.io
+  - Removal of the v1alpha API
+  - kubectl cert-manager status command to help with investigating issues
+  - Using new and stable Kubernetes APIs
+  - Improved logging
+  - ACME improvements
+  - kubectl cert-manager create certificaterequest for signing local certificates
+  - General Availability of JKS and PKCS&#35;12 keystore support
+  - kubectl cert-manager CLI plugin allowing manual renewal and API version conversion
+  - ACME External Account Binding support
+  - Support for full set of x509 ‘subject’ parameters ([#542](https://github.com/mesosphere/kubernetes-base-addons/pull/542), [@jr0d](https://github.com/jr0d))
+  - The Deployment selectors were changed, use `delete` `upgrade-strategy`.
+  - support being upgraded from v0.10 to v1.0.3. ([#594](https://github.com/mesosphere/kubernetes-base-addons/pull/594), [@jr0d](https://github.com/jr0d))
+  - support upgrading from v0.10.1 to v1.0.3 ([#656](https://github.com/mesosphere/kubernetes-base-addons/pull/656), [@jr0d](https://github.com/jr0d))
```

Comment:
these two seem to be saying the same thing, can combine and just put the two PRs at the end?